cylc/flow/run_modes/__init__.py

```diff
@@ -85,6 +85,14 @@ def get(options: 'Values') -> "RunMode":
             return RunMode(run_mode)
         return RunMode.LIVE
 
+    @classmethod
+    def _missing_(cls, value):
+        value = value.lower()
+        for member in cls:
+            if member.lower() == value:
+                return member
+        return None
```

Comment:
Surprised to learn about this. The fact that it's "sunder" (single underscore) is weirding me out. https://docs.python.org/3/library/enum.html#enum.Enum._missing_